src/app/modules/main/wallet_section/all_tokens/view.nim

```diff
@@ -158,6 +158,53 @@ QtObject:
     read = getTokenGroupByCommunity
     notify = tokenGroupByCommunityChanged
 
-  proc toggleTokenGroupByCommunity*(self: View) {.slot.} =
-    self.delegate.toggleTokenGroupByCommunity()
-    self.tokenGroupByCommunityChanged()
\ No newline at end of file
+  proc toggleTokenGroupByCommunity*(self: View): bool {.slot.} =
+    if not self.delegate.toggleTokenGroupByCommunity():
+      error "Failed to toggle tokenGroupByCommunity"
+      return
+    self.tokenGroupByCommunityChanged()
+
+  proc showCommunityAssetWhenSendingTokensChanged*(self: View) {.signal.}
+
+  proc getShowCommunityAssetWhenSendingTokens(self: View): bool {.slot.} =
+    return self.delegate.getShowCommunityAssetWhenSendingTokens()
+
+  QtProperty[bool] showCommunityAssetWhenSendingTokens:
+    read = getShowCommunityAssetWhenSendingTokens
+    notify = showCommunityAssetWhenSendingTokensChanged
+
+  proc toggleShowCommunityAssetWhenSendingTokens*(self: View) {.slot.} =
+    if not self.delegate.toggleShowCommunityAssetWhenSendingTokens():
+      error "Failed to toggle showCommunityAssetWhenSendingTokens"
+      return
+    self.showCommunityAssetWhenSendingTokensChanged()
+
+  proc displayAssetsBelowBalanceChanged*(self: View) {.signal.}
+
+  proc getDisplayAssetsBelowBalance(self: View): bool {.slot.} =
+    return self.delegate.getDisplayAssetsBelowBalance()
+
+  QtProperty[bool] displayAssetsBelowBalance:
+    read = getDisplayAssetsBelowBalance
+    notify = displayAssetsBelowBalanceChanged
+
+  proc toggleDisplayAssetsBelowBalance*(self: View) {.slot.} =
+    if not self.delegate.toggleDisplayAssetsBelowBalance():
+      error "Failed to toggle displayAssetsBelowBalance"
+      return
+    self.displayAssetsBelowBalanceChanged()
+
+  proc displayAssetsBelowBalanceThresholdChanged*(self: View) {.signal.}
+
+  proc getDisplayAssetsBelowBalanceThreshold(self: View): QVariant {.slot.} =
+    return newQVariant(self.delegate.getDisplayAssetsBelowBalanceThreshold())
+
+  proc setDisplayAssetsBelowBalanceThreshold(self: View, threshold: QVariant) {.slot.} =
+    if not self.delegate.setDisplayAssetsBelowBalanceThreshold(threshold.int64Val()):
```

Comment:
It still expects an `int64` here?

Reply:
Alright... the point is to use a `double` property, both in QML and in NIM; not sure if the latter is possible though :/ A regular `float` won't have the required precision for sure